packs/travisci/actions/get_branches.py

```diff
@@ -0,0 +1,18 @@
+from lib.action import TravisCI
+import requests
+import yaml
+
+
+class GetBranches(TravisCI):
```

Comment:
Minor thing - for consistency reasons, every action class should have `Action` suffix, so `ListBranchesAction`.

It's also better to use `List` instead of `Get`. We usually use get when you retrieve a single instance and list when you retrieve more.